<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>541</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>602</width>
      <height>600</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="handleWidth">
     <number>2</number>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>293</height>
       </rect>
      </property>
      <attribute name="label">
       <string>QListWidget操作</string>
      </attribute>
      <widget class="QToolButton" name="tBtnListIni">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>初始化列表</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListClear">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>清除列表</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListInsert">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>插入项</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListAppend">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>添加项目</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListDelete">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>删除当前项</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>293</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>800</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>QListWiget</string>
      </attribute>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>315</width>
         <height>231</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnSelectItem">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>项选择</string>
       </property>
       <property name="autoRepeatDelay">
        <number>298</number>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnSelAll">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnSelNone">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>全不选</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnSelInvs">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>反选</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>当前项变化</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkBoxListEditable">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>可编辑</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListIni"/>
   <addaction name="actListInsert"/>
   <addaction name="actListDelete"/>
  </widget>
  <action name="actListIni">
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actListInsert">
   <property name="text">
    <string>插入项目</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="text">
    <string>添加项</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="text">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="text">
    <string>项选择</string>
   </property>
  </action>
  <action name="actSelAll">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actListClear</sender>
   <signal>triggered()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
